pkg/namespacestore/namespacestore.go

```diff
@@ -632,8 +629,8 @@ func RunReconcile(cmd *cobra.Command, args []string) {
 	}
 	namespaceStoreName := args[0]
 	klient := util.KubeClient()
-	intervalDuration := time.Duration(3)
-	util.Panic(wait.PollImmediateInfinite(intervalDuration*time.Second, func() (bool, error) {
+	intervalSec := time.Duration(3)
```

Comment:
Referring to Guy's suggestion on the previous PR:
https://github.com/noobaa/noobaa-operator/pull/646#discussion_r651707785

Reply:
Reverted to the original, unrelated to the current issues.